src/Pager/Pager.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: rix
+ * Date: 2015.12.21.
+ * Time: 12:44
+ */
+
+namespace Bolt\Pager;
```

Comment:
Can you nuke these, or turn them into proper PHPDoc class blocks just above the class declaration, please :+1: